packages/flutter_tools/lib/src/commands/channel.dart

```diff
@@ -51,11 +51,30 @@ class ChannelCommand extends FlutterCommand {
     }
   }
 
-  Future<void> _listChannels({ bool showAll, bool verbose }) async {
+  String _formatChannels(String currentBranch, String channel) =>
```

Comment:
Any methods that go over a single line should use a block for the body instead of an `=>`.